app/models/nav.rb

```diff
@@ -17,7 +17,13 @@ def current_item_root(request)
   def current_item(request)
     return nil if request.path == "/docs"
 
-    item = route_map[request.path.sub("/docs/", "")]
+    # Temporary workaround to release hidden pages without a nav link
+    path = request.path
+    if path =~ Regexp.new("/docs/apis/rest-api/analytics/(?!flaky-tests)")
+      path = "/docs/apis/rest-api"
+    end
+
+    item = route_map[path.sub("/docs/", "")]
```

Comment:
This prevents the navigation bar code from raising and exception when there is no navigation link for this hidden page.

Our team has a plan to add the nav links for hidden pages once the feature is generally available to all TA customers. We will remove this workaround at upon the marketing release. 

cc @dannymidnight

Reply:
Internal plan regarding a release plan for TA endpoint docs. https://3.basecamp.com/3453178/buckets/32646751/messages/6225392969

Reply:
^ I've left a comment on the Basecamp post to discuss further.

Reply:
We've reverted this change and will fallback to merging and releasing when the feature is ready instead.